apollo-ios/Sources/Apollo/ApolloClientProtocol.swift

```diff
@@ -54,12 +54,14 @@ public protocol ApolloClientProtocol: AnyObject {
   /// - Parameters:
   ///   - mutation: The mutation to perform.
   ///   - publishResultToStore: If `true`, this will publish the result returned from the operation to the cache store. Default is `true`.
+  ///   - contextIdentifier: [optional] A unique identifier for this request, to help with deduping cache hits for watchers. Should default to `nil`.
   ///   - context: [optional] A context that is being passed through the request chain. Should default to `nil`.
   ///   - queue: A dispatch queue on which the result handler will be called. Should default to the main queue.
   ///   - resultHandler: An optional closure that is called when mutation results are available or when an error occurs.
   /// - Returns: An object that can be used to cancel an in progress mutation.
   func perform<Mutation: GraphQLMutation>(mutation: Mutation,
                                           publishResultToStore: Bool,
+                                          contextIdentifier: UUID?,
```

Comment:
Hey guys, this is actually a breaking change that has been introduced between 1.9.0 and 1.9.1

My team who used with 1.9.0 earlier this week is having issues pushing internal libraries using Cocoapods `~> 1.9.0` because it grabs the newer version 1.9.1 and classes that were using `ApolloClientProtocol` now aren't able to compile due to the missing `contextIdentifier`

Reply:
Ooof. Yeah we missed this. Sorry for the difficulties that caused. We'll need to roll this out as a minor version bump at least and put out a new 1.9 patch that reverts this change. I'll handle that tomorrow morning.

Reply:
e.g.: if we were using dependency injection to inject a `ApolloClientProtocol` to a class or struct, it would now not compile in 1.9.1 because `ApolloClientProtocol` now has `contextIdentifier` as an additional parameter.

This was probably not intended, maybe something like this would prevent this to be a breaking change in the future:
```swift
public extension ApolloClientProtocol {
  func perform<Mutation: GraphQLMutation>(
    mutation: Mutation,
    publishResultToStore: Bool,
    context: RequestContext?,
    queue: DispatchQueue,
    resultHandler: GraphQLResultHandler<Mutation.Data>?
  ) -> Cancellable {
    perform(
      mutation: mutation,
      publishResultToStore: publishResultToStore,
      contextIdentifier: nil,
      context: context,
      queue: queue,
      resultHandler: resultHandler
    )
  }
}
```
Otherwise a major version bump would have been expected.

Reply:
I've release 1.9.2 with your suggested change here. Thanks @michaelloo

Reply:
Thanks @AnthonyMDev for the quick fix!